includes/abstract/feedzy-rss-feeds-admin-abstract.php

```diff
@@ -552,6 +556,11 @@ public function feedzy_lazy_load( $data ) {
 		$atts     = $data['args'];
 		$sc       = $this->get_short_code_attributes( $atts );
 		$feed_url = $this->normalize_urls( $sc['feeds'] );
+		$nonce    = isset( $atts['nonce'] ) ? $atts['nonce'] : '';
+
+		if ( ! wp_verify_nonce( $nonce, 'https://codeinwp.test/proof.xml' ) ) {
+			wp_send_json_error( array( 'message' => __( 'Security check failed.', 'feedzy-rss-feeds' ) ) );
```

Comment:
let s use an existing string.

Reply:
This string is already used [here](https://github.com/Codeinwp/feedzy-rss-feeds/blob/master/includes/admin/feedzy-rss-feeds-admin.php#L3059)